pkg/gardenlet/apis/config/v1alpha1/defaults.go

```diff
@@ -340,6 +367,9 @@ func SetDefaults_SNIIngress(obj *SNIIngress) {
 	}
 
 	if obj.Labels == nil {
-		obj.Labels = map[string]string{"istio": "ingressgateway"}
+		obj.Labels = map[string]string{
+			v1beta1constants.LabelApp: "istio-ingressgateway",
+			"istio":                   "ingressgateway",
```

Comment:
Why is the label `"istio": "ingressgateway"` still used here while it is deleted in the other places?

Reply:
Same as above.